src/radio-group/middleware.ts

```diff
@@ -0,0 +1,34 @@
+import { create } from '@dojo/framework/core/vdom';
+import { createICacheMiddleware } from '@dojo/framework/core/middleware/icache';
+
+interface RadioGroupICache {
+	initial: string;
+	value: string;
+}
+
+const icache = createICacheMiddleware<RadioGroupICache>();
+const factory = create({ icache });
+
+export const radioGroup = factory(({ middleware: { icache } }) => {
+	return (onValue: (value: string) => void, initialValue: string) => {
+		const existingInitialValue = icache.get('initial');
```

Comment:
I think this should be typed as a string: `icache.get<string>('initial');`?

Reply:
it's typed above @JamesLMilner via `RadioGroupICache`